### ISSUE TITLE
Document benchmark.json

### ISSUE THREAD

**Issue by arahuja** [hammerlab/variant-calling-benchmarks#17]
Is there some documentation on what `benchmark.json` should be?

By inspection, it seems that one benchmark to correspond to one file of validated variants?  It can support multiple patients, but gold-standard variants should be in a single file?

**Comment by timodonnell** [comment]
Yeah, good question. I've added some more info to the README here (as part of PR #19), interested if this clears it up: https://github.com/hammerlab/variant-calling-benchmarks/blob/917ecb543e5d68054ce5f5b799549a28ada6d4d3/README.md
